index.html

```diff
@@ -0,0 +1,117 @@
+                <img class="logo" src="./img/logo/logo.png" width="153">
+            </a>
+            <a href ="/login" class="loginbutton" class="button">로그인</a>
+        </div>
+        
+    </header>
+    <main>
+        <section class="banner">
+            <div class="banner-wrapper">
+                <div class="banner-textbox">
+                    <h1>일상의 모든 물건을<br>
+                        거래해 보세요
+                    </h1>
+                    <a href="/items" class="itemsbutton" class="button">구경하러 가기</a>
+                </div>
+                <img class="img_home_top" src="./img/Img_home_top.png" width="746">
+            </div>
+        </section>
+        <section class="section1">
+            <div class="section-wrapper">
+                <img class="Img_home_01" src="./IMG/Img_home_01.png" width="558">
+                <div class="section-textbox">
+                    <p class="tag">Hot Item</p>
+                    <h1>인기 상품을<br>
+                    확인해 보세요
+                    </h1>
+                    <h3>가장 HOT한 중고거래 물품을<br>
+                        판다 마켓에서 확인해 보세요
+                    </h3>
+                </div>
+                
+            </div>
+        </section>
+        <section class="section2">
+            <div class="section-wrapper">
+                <div class="section-textbox">
+                    <p class="tag">Search</p>
+                    <h1>구매를 원하는<br>
+                    상품을 검색하세요
+                    </h1>
+                    <h3>구매하고 싶은 물품은 검색해서<br>
+                        쉽게 찾아보세요
+                    </h3>
+                </div>
+                <img class="Img_home_02" src="./IMG/Img_home_02.png" width="588">
+            </div>
+        </section>
+        <section class="section3">
+            <div class="section-wrapper">
+                <img class="Img_home_03" src="./IMG/Img_home_03.png" width="588">
+                <div class="section-textbox">
+                    <p class="tag">Register</p>
+                    <h1>판매를 원하는<br>
+                    상품을 등록하세요
+                    </h1>
+                    <h3>어떤 물건이든 판매하고 싶은 상품을<br>
+                        쉽게 등록하세요
+                    </h3>
+                </div>
+            </div>
+        </section>
+        <section class="bottom">
+            <div class="banner-wrapper">
+                <div class="banner-textbox">
+                    <h1>믿을 수 있는<br>
+                        판다마켓 중고 거래
+                    </h1>
+                </div>
+                <img class="img_home_bottom" src="./img/Img_home_bottom.png" width="746">
+            </div>
+        </section>
+    </main>
+    <footer>
+        <div class="footer-wrap">
+            <a class="copyright">©codeit - 2024</a>
+            <div class="footer-center">
+                <a href="/privacy" class="privacypolicy">Privacy Policy</a>
+                <a href="/faq" class="FAQ">FAQ</a>
+            </div>
+            <div class="logo">
```

Comment:
P3:
더 적절하고 의미가 있는 클래스명이면 좋을 것 같아요~
꼭 해당 페이지의 로고처럼 느껴집니다.

```suggestion
            <div class="sns-wrap">
```